development/request_guidelines.md

```diff
@@ -1,3 +1,21 @@
+# Pull Request Expectations
+
+We have a set of norms around pull requests and merging of code. Most of these have evolved in an informal way among development teams or during our retrospectives while discussing things we could improve. There are exceptions to many rules, but this document attempts to identify our baseline expectations.
+
+## For Submitters
+
+- When containers are modified as a part of a pull request manifests should be updated to use those containers. Alternatively, instructions may be provided to inform reviewers how to deploy the change.
+- All non-trivial pull requests are expected to have a set of testing manual instructions with them. The more information you can provide reviewers about how to verify the change the more likely you are to get good reviews.
+- Pull requests should be merged by the person who submitted them. We leave them open for the submitter so they may do minor cleanup before merging and control the order of merges if they have multiple PRs open.
+- Pull requests should never be merged without a review. We require 1 review for smaller pull requests and at least 2 for larger pull requests. There's no defined line between "small" and "large" pull requests, and that is left to the submitter's discretion. The more reviews you get the less likely you are to be introducing bugs or nonworking code. As a general rule, we value stability over speed when it comes to introducing features.
+- All repositories are expected to use our [build tools](https://github.com/drud/build-tools).
+
+## For Reviewers
+
+- Avoid reviewing code you wrote if possible. Sometimes this cannot be avoided as we have small teams and multiple people working on the same features at times.
+- We have no dedicated QA team. All reviews are expected to include manual testing/QA when appropriate. If manual testing instructions are not provided in the PR and you are unsure of how to test, please ask the submitter to provide them.
+
+
 # Request Guidelines
 A key tenant of any successful software project is communication.  If you are creating an issue or a pull request, you will have a greater chance for success by following this guide.
```

Comment:
Suggestion: While we encourage PRs, we suggest starting with a filed GitHub issue first. This allows us identify a related or duplicate issue. It also provides an opportunity to save the submitted time if another approach would be more in alignment with the projects overall objectives. We want to be respectful of your efforts, and few things are more disheartening than a rejected PR.

Reply:
This and the CLA suggestion sort of highlights a balancing act we're trying to strike with these documents. We put them in public and sort of try to pretend they are for everyone, but the reality is they largely are for our team only.

I think it's great to have these in public and be transparent about our processes, but it doesn't seem reasonable to expect potential contributors to read this repository. If we want to enforce norms in our open source work I feel like we need to work on a distilled document with just the important stuff.

Reply:
I added, "It is strongly suggested that automated tests and documentation updates are included in pull requests." to reflect these aspirations :)